### DISCUSSION TITLE
login in react and flask-socketio

### DISCUSSION THREAD

**Discussion by 6273parastu** [miguelgrinberg/Flask-SocketIO#1904] [Q&A]
login in flask-socketio and react need token_required and token base authentication such as JWT , i search example code and i don`t found it, please help me, tnx.

**Comment by miguelgrinberg** [comment]
There is some information on how to implement authentication in the documentation: https://flask-socketio.readthedocs.io/en/latest/implementation_notes.html#authentication. Unfortunately I do not have any examples to share.
